infra/auth0/main.tf

```diff
@@ -61,6 +61,10 @@ resource "auth0_branding" "branding" {
     # online-blue
     page_background = "#0D5474"
   }
+
+  universal_login {
```

Comment:
Curious about this, what does it do

Reply:
This is the easter background, I just forgot to push it

Reply:
Nice! And creds for the commits, really pleasant to review